### ISSUE TITLE
Add models for storing buyers and sellers

### ISSUE THREAD

**Issue by mimmon** [ricco386/django-faktura#12]
If multiple invoices for the same buyer are issued, it will be handy to have them stored in the database. 
The same is true for seller which will most probably will usually be the only one.
These models can be separate and inherited from common abstract InvoiceUserModel, but this can be discussed (we can only use a single model)

**Comment by ricco386** [comment]
Yes this is true to a certain point. Both buyer and seller cound change the data and the old invoices have to stay the same... For example if the buyer moves he would have different address and only new invoices should be amended.

Currently we have  the duplicate function so it also creates an invoice with the details so there is less work.

We can achieve this behaviour in DB as well.
